### ISSUE TITLE
games updates showing zero size

### ISSUE THREAD

**Issue by voidplayer** [ValveSoftware/steam-for-linux#4020]
![selection_164](https://cloud.githubusercontent.com/assets/4019910/9906005/731d7862-5c88-11e5-8188-6fdaf24154ee.png)

It only get the size when it starts downloading. Not sure if is linux specific, but it used to work.

**Comment by Tele42** [comment]
This has been a long-standing quirk with no functional side-effect, download size shows the size of the last download when it's in the queue.

**Comment by voidplayer** [comment]
yeah, its merely cosmetic... it works sometimes tho... thats why i classify it as a bug tho... 

If you are not going to show the size, remove the information in the first place...

Also, the functional side-effect would be the to progress line in the bottom of the client showing inaccurate progress.

Its annoying the fact that it works some times and other it doesnt... there is no way to tell if the progress bar is being accurate or it is not

**Comment by ozone89** [comment]
to add insult to the injury, the disk usage graph doesn't work, either
